release_notes/374-release-notes.adoc

```diff
@@ -330,13 +331,17 @@ If you are using the in-product docs, you can instead download the link:https://
 * The `v1/alerts/summary/counts` endpoint, which is used by the *Policy violations by severity* and *Policy violations by category* widgets on the Dashboard, was fixed to prevent Central from crashing in highly-scaled environments. (ROX-13829)
 * {product-title-short} previously was not able to correctly identify the signature of images that contained the same content but were stored in different registries, causing problems with alerts and enforcement. This issue has been fixed and {product-title-short} now correctly identifies signatures of images from different registries. (ROX-12400)
 
+[id="resolved-in-version-3741"]
+=== Resolved in version 3.74.1
+
+* Previously, Sensor on secured clusters could not connect to Central when using a proxy, even if the proxy was specified in the proxy configuration file or in environment variables. This issue is fixed. (ROX-15279)
```

Comment:
Just a suggestion, feel free to disregard some/all
```suggestion
* Previously, secured clusters on Sensor could not connect to Central when using a proxy, even if the proxy was specified in the proxy configuration file or in environment variables. This issue is fixed and Sensor on secured clusters now connects to Central when using a proxy with no issue. (ROX-15279)
```

Reply:
Thanks - I don't think the first sentence is technically accurate but let me reword. Good suggestion for the second!